include/output_modification/convert_output.hpp

```diff
@@ -0,0 +1,121 @@
+/** \brief Output Format Conversion source file
+ *
+ *  Authors      : Abhinav Achanta, Rajeev Jandhyala
+ *  
+ *  This function is used to transform the ABP simulator generated output 
+ *  to a tabular and readable format. It takes the file as input and 
+ *  can transform and store in a new output file provided the name.
+ *   
+ *  
+ *  This function can be used for the formatted output of components
+ *  ABP Simulator, Sender,Receiver, Subnet
+ */
+
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <iomanip>
+
+using namespace std;
+
+/** 
+ * Function to get the data from the input file
+ * and write the data in new format to the output file
+ * @param ip_file_name input file name string
+ * @param op_file_name output file name string
+ */
+void convert_output(char * ip_file_name, char * op_file_name) {
```

Comment:
You need to define only function declaration in .hpp file and definition/implementation  will go in .cpp file.

So it will be like this

1) convert_output.hpp
2) convert_output.cpp

Refer to how message.hpp and message.cpp is implemented